competency-explorer/src/selectors/visibility.ts

```diff
@@ -0,0 +1,77 @@
+import { State, Level, CompetencyId, Category } from "../models";
+import {
+  selectCompetency,
+  selectCompetenciesForCategory,
+  selectCompetenciesForLevelAndCategory,
+} from "./competencies";
+import { selectCategories } from './categories';
+import { selectLevels } from './levels';
+
+export const HIDDEN = false;
+export const VISIBLE = true;
```

Comment:
Wishing for https://github.com/microsoft/TypeScript/issues/15807 here 🙏

Reply:
🙏